src/components/containers/menu/MenuSection.js

```diff
@@ -21,11 +21,11 @@ const StyledChildrenContainer = styled.div`
 `;
 
 const MenuSection = (props, context) => {
-  const { title, children, isLast, isFirst } = props;
+  const { title, children, isLast, isFirst, theme } = props;
   const { inline } = context;
 
   return (
-    <ThemeProvider theme={defaultTheme}>
+    <ThemeProvider theme={theme}>
```

Comment:
Again, if MenuSection will always have a Menu parent somewhere in the component tree, you shouldn't need the ThemeProvider or defaultTheme here. If MenuSection is intended for stand-alone use, then it can keep the ThemeProvider.